Jenkinsfile

```
@@ -40,4 +40,6 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipeline(['distros' : ['centos7', 'centos8', 'leap15']])
\ No newline at end of file
+packageBuildingPipeline(['distros' : ['el8', 'leap15']])
+
+packageBuildingPipelineDAOSTest(['distros' : ['el8', 'leap15'], 'test-tag': 'vol'])
```

You only want one of these:

```suggestion
packageBuildingPipelineDAOSTest(['distros' : ['el8', 'leap15'], 'test-tag': 'vol'])
```

---

good catch. i missed removing that line